website/docs/patterns/data-visualization/partials/guidelines/color-usage.md

```diff
@@ -0,0 +1,46 @@
+## Color usage
+
+When creating data visualizations, care should be taken to select colors geared towards the type of data being presented. General color palettes created for web UIs and/or marketing purposes are not always suitable as the colors can lack contrast and distinguishability or else be visually unsuitable when used together in a data context. Keep in mind that color choice can also impact meaning.
+
+**There are three main types of color palettes used for data visualization:**
+
+1. Categorical palettes
+2. Sequential palettes
+3. Diverging palettes
+
+### Categorical palettes
+
+Use for presenting categorical data that has no intrinsic order, such as a pie or donut chart. Also used for multi-line charts, area charts, stacked or grouped bar charts, etc.
+
+**Categorical palettes should:**
+
+* Use distinct colors.
+* Be limited to a max of 10 colors.
+* Be distinguished primarily by hue.  
+(**Important:** For accessibility, use patterns, symbols, or text in addition to or instead of color.)
```

Comment:
Does the gap/border between the colors help with this at all?

Reply:
It does somewhat so I can explicitly call this out. Thanks for mentioning.

Reply:
I added mention of this above in the "Accessiblity considerations" section